src/metabase/util.clj

```diff
@@ -472,6 +472,13 @@
      ~@body
      ~'<>))
 
+(defn format-emoji
+  "Returns the string passed in if emojis in logs are enabled, an empty string otherwise."
+  [format-string]
+  (if (config/config-bool :mb-emoji-in-logs)
+      format-string
+      ""))
```

Comment:
You can take this one step further and do only do the `if` check once when the app starts instead of doing it every time the function is called (it's not going to change during runtime after all). 

```clojure
(def ^String ^{:arglists '([emoji-string])} format-emoji
  "Returns the EMOJI-STRING passed in if emoji in logs are enabled, otherwise always returns an empty string."
  (if (config/config-bool :mb-emoji-in-logs)
    identity
    (constantly "")))
```

This is similar to what we do with `format-color`